.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: java
 
 jdk:
-  - oraclejdk7
```

Comment:
Are you sure? http://www.draconianoverlord.com/2014/04/01/jdk-compatibility.html